apps/web/pages/auth/login.tsx

```diff
@@ -39,7 +39,6 @@ interface LoginValues {
   csrfToken: string;
 }
 export default function Login({
-  csrfToken,
```

Comment:
I know we talked briefly about this but even after digging in I still don't understand why we're removing the csrf token in this PR. That feels like it has further reaching consequences that are unrelated to adding a cc. Can you either explain in more detail why this is necessary or remove it?

Reply:
if you run 'yarn dev' it will break because of the cfsf token, which is not used. it was used in the code for login with password but that is commented out. thi code is live on stage and it works. it's ok. 👍🏼

Reply:
ah got it, makes sense